<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSetPrm</class>
 <widget class="QDialog" name="DlgSetPrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка приёмника</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupStrobe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Стробы</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkGsh">
        <property name="text">
         <string>ГШ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkZu">
        <property name="text">
         <string>ЗУ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkPilot">
        <property name="text">
         <string>ПС</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Код ШАРУ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCh1">
        <property name="text">
         <string>Канал СУММ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinCh1">
        <property name="maximum">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCh2">
        <property name="text">
         <string>Канал РАЗН</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinCh2">
        <property name="maximum">
         <number>31</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkAllow">
       <property name="text">
        <string>Разрешение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBlock">
       <property name="text">
        <string>Блокировка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBlock</tabstop>
  <tabstop>checkGsh</tabstop>
  <tabstop>checkZu</tabstop>
  <tabstop>checkPilot</tabstop>
  <tabstop>spinCh1</tabstop>
  <tabstop>spinCh2</tabstop>
  <tabstop>pushClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
